doc/src/tutorials/biomechanics/biomechanics.rst

```diff
@@ -0,0 +1,1532 @@
+as activation dynamics. Because activation dynamics are so common in
+biomechanical models, SymPy provides the
+:obj:`~sympy.physics.biomechanics.activation` module, which contains
+implementations for some common models of activation dynamics. These are
+zeroth-order activation dynamics and first-order activation dynamics based on
+the equations from the paper by [DeGroote2016]_. Below we will work through
+manually implementing these models and then show how these relate to the classes
+provided by SymPy.
+
+Zeroth-Order
+------------
+
+The simplest possible model of activation dynamics is to assume that diffusion
+of :math:`\textrm{Ca}^{2+}` ions is instantaneous. Mathematically this gives us
+:math:`a(t) = e(t)`, a zeroth-order ordinary differential equation.
+
+>>> e = me.dynamicsymbols('e')
+>>> e
+e(t)
+>>> a = e
+>>> a
+e(t)
+
+Alternatively, you could give :math:`a(t)` its own
+:obj:`~sympy.physics.vector.dynamicsymbols` and use a substitution to replace
+this with :math:`e(t)` in any equation.
+
+>>> a = me.dynamicsymbols('a')
+>>> zeroth_order_activation = {a: e}
+>>> a.subs(zeroth_order_activation)
+e(t)
+
+SymPy provides the class
+:obj:`~sympy.physics.biomechanics.activation.ZerothOrderActivation` in the
+:obj:`~sympy.physics.biomechanics.activation` module. This class must be
+instantiated with a single argument, `name`, which associates a name with the
+instance. This name should be unique per instance.
+
+>>> from sympy.physics.biomechanics import ZerothOrderActivation
+>>> actz = ZerothOrderActivation('zeroth')
+>>> actz
+ZerothOrderActivation('zeroth')
+
+The argument passed to `name` tries to help ensures that the
+automatically-created :obj:`~sympy.physics.vector.dynamicsymbols` for
+:math:`e(t)` and :math:`a(t)` are unique betweem instances.
+
+>>> actz.excitation
+e_zeroth(t)
+>>> actz.activation
+e_zeroth(t)
+
+:obj:`~sympy.physics.biomechanics.activation.ZerothOrderActivation` subclasses
+:obj:`~sympy.physics.biomechanics.activation.ActivationBase`, which provides a
+consistent interface for all concrete classes of activation dynamics. This
+includes a method to inspect the ordinary differential equation(s) associated
+with the model. As zeroth-order activation dynamics correspond to a zeroth-order
+ordinary differential equation, this returns an empty column matrix.
+
+>>> actz.rhs()
+Matrix(0, 1, [])
+
+First-Order
+-----------
+
+In practice the diffusion and concentration increase of
+:math:`\textrm{Ca}^{2+}` ions is not instantaneous. In a real biological
+muscle, a step increase in excitation will lead to a smooth and gradual
+increase in activation. [DeGroote2016]_ model this using a first-order
+ordinary differential equation:
+
+.. math::
+
+   \begin{align*}
+      \frac{da}{dt} &= \left( \frac{1}{\tau_a \left(1 + 3a(t)\right)} (1 + 2f) + \frac{1 + 3a(t)}{4\tau_d} (1 - 2f) \right) \left(e(t) - a(t) \right) \\
+      f &= \frac{1}{2} \tanh{\left(b \left(e(t) -a(t)\right)\right)}
+   \end{align*}
```

Comment:
You don't need to put an align environment around equations, that is done automatically by sphinx's math directive.

Reply:
See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#math